python/semantic_kernel/connectors/mcp.py

```diff
@@ -228,10 +232,6 @@ async def connect(self) -> None:
             # If the session is not initialized, we need to reinitialize it
             await self.session.initialize()
         logger.debug("Connected to MCP server: %s", self.session)
-        with suppress(Exception):
-            logger.debug("Resources: %s", await self.session.list_resources())
-        with suppress(Exception):
-            logger.debug("Resource templates: %s", await self.session.list_resource_templates())
         await self.load_tools()
         await self.load_prompts()
```

Comment:
This call will still hang if the server returns a 400. Any timeout config will cause an issue in our `add_plugin` code where the kernel function metadata will be missing, and we'll add a blank plugin with no underlying functions.